app/services/food_data_service.rb

```diff
@@ -0,0 +1,11 @@
+class FoodDataService
+
+  def self.search_foods(query)
+    conn = Faraday.new(url: "https://api.nal.usda.gov") do |f|
+      f.params["api_key"] = ENV["FOOD_KEY"]
+    end
```

Comment:
Great job setting up the connection object here. You could abstract this functionality into its own method to keep things more SRP.